services/scan/src/util/workspace.ts

```diff
@@ -62,8 +62,8 @@ export function createWorkspace(root: string): Workspace {
     scannedImagesPath,
     uploadsPath,
     store,
-    zero() {
-      store.zero();
+    resetElectionSession() {
```

Comment:
@jonahkagan is this how you envisioned a "session"?

Reply:
In terms of what data would be grouped together, yes, more or less. I imagined it as a first-class data type, but @adghayes and I discussed that in another thread and decided to not break out that type for now